demo/Signs.swift

```diff
@@ -635,6 +790,64 @@ extension SignValue {
+            return market.isChina ? Asset.Signs.regulatoryNoMotorcyclesOrMopedsCN : nil
+        case .regulatoryNoMotorVehiclesExceptMotorcycles:
+            return market.isChina ? Asset.Signs.regulatoryNoMotorVehiclesExceptMotorcyclesCN : nil
+        case .regulatoryNoMotorcyclesOrMopeds:
+            return market.isChina ? Asset.Signs.regulatoryNoMotorcyclesOrMopedsCN : nil
+        case .regulatoryNoMotorVehiclesEntry:
+            return market.isChina ? Asset.Signs.regulatoryNoMotorVehiclesEntryCN : nil
+        case .regulatoryNoTurnLeftOrTurnRight:
+            return market.isChina ? Asset.Signs.regulatoryNoTurnLeftOrTurnRightCN : nil
+        case .regulatoryNoOvertaking:
+            return market.isChina ? Asset.Signs.regulatoryNoOvertakingCN : nil
+        case .regulatoryNoHonking:
+            return market.isChina ? Asset.Signs.regulatoryNoHonkingCN : nil
+        case .regulatoryWidthLimit:
+            return nil
+        case .regulatoryAxleWeightLimit:
+            return market.isChina ? Asset.Signs.regulatoryAxleWeightLimitCN : nil
+        case .regulatoryNoVehiclesCarryingExplosives:
+            return market.isChina ? Asset.Signs.regulatoryNoVehiclesCarryingExplosivesCN : nil
+        case .regulatoryRoundabout:
+            return market.isChina ? Asset.Signs.regulatoryRoundaboutCN : nil
+        case .regulatoryHonking:
+            return market.isChina ? Asset.Signs.regulatoryHonkingCN : nil
+        case .regulatoryPedestriansCrossing:
+            return market.isChina ? Asset.Signs.regulatoryPedestriansCrossingCN : nil
+        case .regulatoryMotorVehicles:
+            return market.isChina ? Asset.Signs.regulatoryMotorVehiclesCN : nil
+        case .regulatoryUTurn:
+            return market.isChina ? Asset.Signs.regulatoryUTurnCN : nil
+        case .warningSteepAscent:
+            return market.isChina ? Asset.Signs.warningSteepAscentCN : nil
+        case .warningSteepDescent:
+            return market.isChina ? Asset.Signs.warningSteepDescentCN : nil
+        case .warningVillage:
+            return market.isChina ? Asset.Signs.warningVillageCN : nil
+        case .warningRailroadCrossingWithBarriers:
+             return market.isChina ? Asset.Signs.warningRailroadCrossingWithBarriersCN : nil
+        case .warningNoMotorVehicles:
+            return market.isChina ? Asset.Signs.warningNoMotorVehiclesCN : nil
+        case .warningKeepSlowdown:
+            return market.isChina ? Asset.Signs.warningKeepSlowdownCN : nil
+        case .warningDangerousTraffic:
+            return market.isChina ? Asset.Signs.warningDangerousTrafficCN : nil
+        case .warningRoadworks:
+            return market.isChina ? Asset.Signs.warningRoadworksCN : nil
+        case .warningSecondRoadLeft:
+            return market.isChina ? Asset.Signs.warningSecondRoadLeftCN : nil
+        case .otherSigns:
+            return nil
         }
     }
 }
+
+private extension Market {
+    var isChina: Bool {
```

Comment:
This approach may work while we have 2 countries, but when at least 3rd one is added all the lines which involve `isChina` will be changed. 
It may be easier to see changes and unspecified assets when we're switching by country.
Since we're trying to keep name consistency it should be easy to edit with any multiline editing tool.

Reply:
Fixed in my commit. @vanmaxim PTAL.

Reply:
@chezzdev LGTM